tools/csv-merger/csv-merger.go

```diff
@@ -609,7 +608,7 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 		}
 		ocsCSV.Annotations["createdAt"] = time.Now().In(loc).Format("2006-01-02 15:04:05")
 	}
-	ocsCSV.Annotations["containerImage"] = "quay.io/ocs-dev/ocs-operator:" + ocsversion.Version
+	ocsCSV.Annotations["containerImage"] = *ocsContainerImage
```

Comment:
why is this change required?

Reply:
This is what produced the change in `deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml`. Since it's in an annotation it is largely cosmetic, so there's no strict "requirement" for it. But it's still technically wrong, so why not?